### PR TITLE
Fixed Paybox error status

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/Donation/DonationRequestUtils.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ public function createRetryPayload(Donation $donation, Request $request): array
 
     public function createCallbackStatus(Donation $donation): array
     {
-        $code = self::PAYBOX_STATUSES[$donation->getPayboxResultCode()] ?? self::PAYBOX_UNKNOWN;
+        $code = self::PAYBOX_STATUSES[$donation->getPayboxResultCode()] ?? 'error';
```
Comment (illusionOfParadise) [inline src/Donation/DonationRequestUtils.php]: What about `self::PAYBOX_STATUSES[self::PAYBOX_UNKNOWN]` or creating another constant ?

Comment (HeahDude) [inline src/Donation/DonationRequestUtils.php]: I thought about that, was not sure of the benefit of a new constant, I guess we should go for `self::PAYBOX_STATUSES[self::PAYBOX_UNKNOWN]` then.

Comment (tgalopin) [inline src/Donation/DonationRequestUtils.php]: `self::PAYBOX_STATUSES[self::PAYBOX_UNKNOWN]` seems nice indeed

Comment (HeahDude) [inline src/Donation/DonationRequestUtils.php]: Done.

**Pull request merged**